homeassistant/components/intesishome/climate.py

```diff
@@ -248,21 +296,25 @@ def target_temperature_step(self) -> float:
         self._fan_speed = fan_mode
         self.async_write_ha_state()
 
+    async def async_set_preset_mode(self, preset_mode):
+        """Set preset mode."""
+        ih_preset_mode = MAP_PRESET_MODE_TO_IH.get(preset_mode)
+        await self._controller.set_preset_mode(self._device_id, ih_preset_mode)
+
+        # Updates can take longer than 2 seconds, so update locally
+        self._preset = ih_preset_mode
+        self.async_write_ha_state()
```

Comment:
Remove this. That is wrong: https://developers.home-assistant.io/docs/architecture_index/